src/sidebar/store/modules/selection.js

```diff
@@ -5,10 +5,26 @@
 
 /**
  * @typedef User
- * @property {string} userid - Unique user's id
+ * @property {string} [userid]
```

Comment:
Can you use `@prop` for the property definitions. It means the same as `@property` but is a bit shorter and the more common convention.

Reply:
Ah, I was just c'n'ping blindly here...

Reply:
Intriguingly, my TypeScript editor plugin doesn't recognize `@prop` and does not syntax-highlight for it. Irritating...